### DISCUSSION TITLE
maximum number of rows when fetching from a dataset?

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3897] [Q&A]
hello, when reviewing the documentation I realize that in the triggers section there is an option which limits the amount of data to be consulted through the api, and which generates a limit, it says that more than 20 thousand is not recommended, but If there is a set with 2 million data, what could be an option to bring all that information through the api? And perhaps what could be the technological limitation of that query, thank you very much,

**Comment by janette** [comment]
@tridoxx you will need to conduct load testing to discover the limit your web server can handle.
